frontend/src/libs/components/button/button.tsx

```diff
@@ -1,10 +1,17 @@
 type Properties = {
   label: string;
   type?: 'button' | 'submit';
+  className?: string | undefined;
```

Comment:
```suggestion
  className?: string;
```
? already means it can be 'undefined'

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/d5b95509-304a-4764-b883-8a52fc869b85)
When I'm doing it this way, I get an exception

Reply:
It's strange, can you please restart TS?

Reply:
I tried to restart and it didn't help. At now I commited with className={styles['btn'] as string}, but it's really weird

Reply:
To understand the problem better, do you get this error in the linter as well? Or only in the code editor?

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/19ad1174-c573-468e-ac15-5d6a07baff51)
Yeah, I get this error in the linter and can't commit due to it